### PR TITLE
update token

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by ocefpaf** [comment]
> I tried to re-render for you, but it looks like there was nothing to do.

It should've updated the tokens in the CIs but it looks like the webservices is not OK. Ping @isuruf and @dougalsutherland who probably know more about it than me.

I'm issuing a manual re-render for now.

File: .appveyor.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ environment:
 
   BINSTAR_TOKEN:
     # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
-    secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+    secure: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
```
Comment (ocefpaf) [inline .appveyor.yml]: This is what we missed to get the packages updated. @isuruf did an amazing job of updating all the tokens for Travis-CI and AppVeyor on the `master` branch but we could no propagate that to the branches, so we need to do a few of those manually by just copying the `conda-forge.yaml` from `master` and re-rendering.

**Comment by minrk** [comment]
Python 3.7 builds are failing. Shouldn't they be skipped at this point?

**Comment by ocefpaf** [comment]
> Python 3.7 builds are failing. Shouldn't they be skipped at this point?

I'm sending a few "point" updates to Python 3.7. I forgot to re-render this with vanilla conda-smithy and used my "hacked" version. I'll fix this in a second.

**Comment by ocefpaf** [comment]
Meringing this without waiting for the long AppVeyor queue. We know this is passing and we only want the token to get the packages uploaded.

**Comment by jasongrout** [comment]
Thanks for your work on this!

Does this require me to do another PR to get #52 to build and deploy?

**Comment by jasongrout** [comment]
Great, thanks!

**Pull request merged**